src/bun.js/webcore/response.zig

```diff
@@ -843,7 +883,30 @@ pub const Fetch = struct {
                                 var scheduled_response_buffer = this.scheduled_response_buffer.list;
 
                                 const chunk = scheduled_response_buffer.items;
-
+                                for (response.response_clones.?.items) |other_request| {
```

Comment:
will look at it

Reply:
@cirospaciari readable_stream_ref seams to not be part of the request, can you specify how we should handle this?

Reply:
`readable_stream_ref` is created in the `Response` when we call `response.body.getReader()` after this we basically only update the stream because body would be used, in this case we still need to check and update `response_clones` but we do an early return  here https://github.com/oven-sh/bun/blob/3202486e55370827e9fe957422b4a8b81ee8d9cf/src/bun.js/webcore/response.zig#L991-L1019 and we also probably check the fail case here https://github.com/oven-sh/bun/blob/3202486e55370827e9fe957422b4a8b81ee8d9cf/src/bun.js/webcore/response.zig#L965-L989

We can still call `this.response.get()` because we do not discard this reference even when we get a `readable_stream_ref` in `onReadableStreamAvailable`, streams are a little confusing right now